modules/ximgproc/src/sparse_match_interpolators.cpp

```diff
@@ -57,7 +58,8 @@ struct SparseMatch
 
 bool operator<(const SparseMatch& lhs,const SparseMatch& rhs);
 
-void weightedLeastSquaresAffineFit(int* labels, float* weights, int count, float lambda, SparseMatch* matches, Mat& dst);
+void computeGradientMagnitude(Mat& src, Mat& dst);
+void weightedLeastSquaresAffineFit(int* labels, float* weights, int count, float lambda, const SparseMatch* matches, Mat& dst);
```

Comment:
Please declare local (this file only) functions as "static". Or use anonymous namespace for them.

Reply:
Static has been added.